htdocs/web_portal/controllers/downtime/add_downtime.php

```diff
@@ -71,53 +72,52 @@ function submit(\User $user = null) {
     if($confirmed == true){
         //Downtime is confirmed, submit it
         //$downtimeInfo = unserialize($_REQUEST['newValues']);  // didn't cater for UTF-8 chars
+        /**
+         * If confirmed by an user, submit the details of affected services
+         * and endpoints along with other details for each individual site.
+         */
         $downtimeInfo = json_decode($_POST['newValues'], TRUE);
         $serv = \Factory::getDowntimeService();
+        $downtimeInfo = unsetVariables($downtimeInfo, 'add');
```

Comment:
Why is this being called here?

Reply:
We are waiting for the user to confirm the details that they have entered. Once they click submitted we no longer need few variables like `SERVICE_WITH_ENDPOINTS` which we were using earlier for viewing purpose and to extract the site level details(Site A: Services selected list, and Endpoints list).

Reply:
ah okay, I probably wouldnt have bothered explicitly unsetting variables, but I can see it's already done elsewhere around adding and editing downtimes.